pmd-core/src/main/java/net/sourceforge/pmd/renderers/XMLRenderer.java

```diff
@@ -152,6 +152,17 @@ public void end() throws IOException {
                 writer.write(buf.toString());
             }
         }
+        
+        // config errors
+        for (final Report.ConfigurationError ce : configErrors) {
+            buf.setLength(0);
+            buf.append("<configerror ").append("rule=\"");
```

Comment:
Yes, I'm fine with breaking compatibility. We just need to be aware of that.
Creating a schema probably makes sense.